### PR TITLE
configuring linter

### PR DESCRIPTION
**Pull request by dgsierra**
following the instruction for adding the linter.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 h1 {
-    color: red;
-}
\ No newline at end of file
+  color: red;
```
Comment (vechicin) [inline styles.css]: [OPTIONAL] It is considered bad practice to use color names. Consider changing it to its `hex` or `rgb` equivalent

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @dgonzalesi ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good PR title/description
- [x] Proper linters setup

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/dgonzalesi/hello-microverse/blob/074eb27bfb8f093235d089fe37507e6fea690d55/.gitignore#L1-L2
Please add the `node_modules` folder to the `.gitignore` file.

Screen Shot 2022-06-28 at 17 05 06

**Comment by vechicin** [comment]
https://github.com/dgonzalesi/hello-microverse/blob/074eb27bfb8f093235d089fe37507e6fea690d55/README.md?plain=1#L13-L15

Please provide detailed instructions on how to get a local copy of this project.

**Review by topeogunleye — APPROVED** [review]
### Hi @dgonzalesi, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**